### ISSUE TITLE
When MIB polling is active adding a device via API causes weird results returned.

### ISSUE THREAD

**Issue by spinza** [librenms/librenms#3723]
This is what's returned:
```
MIB: registering
     CISCO-AAA-SESSION-MIB::ciscoAAASessionMIB

{
    "status": "ok",
    "message": "Device my.device.com has been added successfully"
}
```
It's not proper JSON.

**Comment by laf** [comment]
This is because register_mibs() isn't using d_echo and simply just echoing out information.

**Comment by spinza** [comment]
Are you referring to the code in this function? 
https://github.com/librenms/librenms/blob/692c29cbd8d7a0d704c85cb055e0b859b514b65c/includes/snmp.inc.php#L1242

**Comment by spinza** [comment]
If so can I replace that in a PR with d_echo?

**Comment by paulgear** [comment]
Should be fixed in #3739

**Comment by spinza** [comment]
Confirmed fixed by #3739

**Comment by spinza** [comment]
Confirmed fixed by #3739
